### PR TITLE
feat: uses new model

### LINKED ISSUE #16: Chore: update ceramic model in UI

### PR DESCRIPTION
**Pull request by fabianschu**
closes #16

### CODE REVIEW

File: packages/next/pages/contribute.tsx [hunk 1]
```diff
@@ -47,54 +48,56 @@ const contribute = () => {
...
+      const newContributionTile = await dataModel.createTile(
+        'Contribution',
+        newContribution
+      )
+      const existingContributions = await dataStore.get('contributions')
+      const oldRecord = existingContributions
+        ? existingContributions.contributions
+        : []
+      const newRecord = {
+        contributions: [...oldRecord, { id: newContributionTile.id.toUrl() }],
```
Comment (Zitzak) [inline packages/next/pages/contribute.tsx:82]: Why did you remove the `title` member when adding new entries to the record?

File: packages/next/pages/contribute.tsx [hunk 2]
```diff
@@ -111,10 +114,16 @@ const contribute = () => {
     setDaoAddress(daoTextField)
   }
 
-  const getDaoContributions = (contributions) =>
-    contributions.filter((contribution) => {
+  const getDaoContributions = (contributions) => {
+    console.log('kek')
```
Comment (Zitzak) [inline packages/next/pages/contribute.tsx:118]: Still some console logs. Here and some lines below

**Review by codebuster22 — APPROVED** [review]
LGTM as far as my understanding.
I'm not sure on the logical changes made. My review have been purely reviewing JSX code.
I think the final review should be done by @Zitzak
